torch/utils/data/dataloader.py

```diff
@@ -81,6 +85,7 @@ class DataLoader(object):
         collate_fn (callable, optional): merges a list of samples to form a
             mini-batch of Tensor(s).  Used when using batched loading from a
             map-style dataset.
+        collate_fn_args (optional): parameters that can be used by collate_fn
```

Comment:
Why is this needed? Can't you just use a `collate_fn` with these args? E.g., `functools.partial`?

Reply:
`collate_fn` is being used to transform batches before converting them to Tensors and return. Some transformations require parameters.

I considered using `functors.partial` to wrap arguments on collate_fn, but I wasn't sure whether that would be the simpler solution for the users.

If you think so, I can do this change

Reply:
Yes, we should not be in the business of managing that. If someone needs extra parameters, they can either make a closure, or use `functools.partial`.

Reply:
Or [use a class with a `__call__` method](https://github.com/facebookresearch/maskrcnn-benchmark/blob/55796a04ea770029a80cf5933cc5c3f3f6fa59cf/maskrcnn_benchmark/data/collate_batch.py#L5-L20)

Reply:
Agreed.